content/team/_index.md

```diff
@@ -73,10 +77,17 @@ SkyWalking have hundreds of Contributors, you could find them in our repository
 - [SkyWalking kubernetes](https://github.com/apache/skywalking-kubernetes/graphs/contributors)
 - [SkyWalking K8s Operator - SkyWalking Cloud on Kubernetes](https://github.com/apache/skywalking-swck/graphs/contributors)
 - [SkyWalking Python Agent](https://github.com/apache/skywalking-python/graphs/contributors)
+- [SkyWalking CLI](https://github.com/apache/skywalking-cli/graphs/contributors)
+- [SkyWalking Client JS](https://github.com/apache/skywalking-client-js/graphs/contributors)
+- [SkyWalking satellite](https://github.com/apache/skywalking-satellite/graphs/contributors)
+- [SkyWalking agent test tool](https://github.com/apache/skywalking-agent-test-tool/graphs/contributors)
+- [SkyWalking NodeJS Agent](https://github.com/apache/skywalking-nodejs/graphs/contributors)
+- [SkyWalking Rust Agent](https://github.com/apache/skywalking-rust/graphs/contributors)
+- [SkyWalking oal tool](https://github.com/apache/skywalking-oal-tool/graphs/contributors)
```

Comment:
I think we should request to achieve this. @apache/skywalking-committers , what do you think?
@fgksgf feel free to submit a JIRA ticket to the INFRA.

> From Nov. 6th 2018, through OAL tool module established PR. This tool has been integrated into SkyWalking main repostory.

Reply:
> I think we should request to achieve this. @apache/skywalking-committers , what do you think?

+1

Reply:
@fgksgf You could move this to `Archived` group.

Reply:
> @fgksgf You could move this to `Archived` group.

Both of `rust` and `OAL tool` ? Or only `OAL tool` ?

Reply:
I think that statement exists for a long time. The vote is harmless, if you want, put one on the dev mail list.

Reply:
> Both of rust and OAL tool ? Or only OAL tool ?

Only OAL tool. Rust is WIP and not active, let's keep it for now.